src/online2/online-ivector-feature.cc

```diff
@@ -67,6 +67,15 @@ void OnlineIvectorExtractionInfo::Init(
   this->Check();
 }
 
+int32 OnlineIvectorExtractionInfo::ExpectedFeatureDim() const {
+  int32 num_splice = 1 + splice_opts.left_context + splice_opts.right_context,
+      full_dim = lda_mat.NumCols();
+  if (!(full_dim % num_splice == 0) || (full_dim % num_splice == 1)) {
```

Comment:
Should be: `if (!(full_dim % num_splice == 0 || full_dim % num_splice == 1)) {`

Reply:
Thanks !!! Fixing